pages/wg/outreach.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: page
+title: Outreach Working Group
+subtitle:
+permalink: /wg/outreach/
+---
+
+# Overview
+
+The US-RSE Outreach Working Group is dedicated to broadening the US-RSE's
+engagement with the RSE and wider computationl science and engineering (CSE)
+communities. We strive to raise awareness of the US-RSE as an association and
+RSE as a professional designation through engagement with industry, academia,
+national labs, and students.
+
+Those interested in contributing to this goal are encouraged to join our
+bi-weekly Teams meetings. To get involved, visit the
```

Comment:
Is "Teams" the technology used?  If so, could we remove it? Otherwise, I think it should just be "team"

Reply:
Teams is the technology used - would you prefer "virtual" meeting?

Reply:
yes